### ISSUE TITLE
[suggestion] Have volume scrolling work on a logarithmic scale

### ISSUE THREAD

**Issue by fxjkhr** [sirWest/AudioSwitch#57]
Manipulating the volume on the lower end with the fine control of the scroll wheel is useful, but on the upper end it becomes tedious since humans perceive sound on a logarithmic scale.  Here is a link that sort of explains what I mean: https://www.reddit.com/r/gamedev/comments/291cft/sound_1_fix_your_volume_sliders_2_experiment_with/

Would it be possible to change the volume with higher difference of "steps" on high volume like Volumouse does? http://imgur.com/uAtRKCj

Basically, the difference between 20% and 1% in volume is bigger than 100% to 80%.  Can the scrolling feature be changed so that the number of scrollwheel steps needed to change from 80-100 is less than 20-1?

**Comment by sirWest** [comment]
this is a good idea in general, I will try to add it where my own volume changing code is used, however, next feature which is almost complete is to optionally be able to use Windows' own OSD for volume also, in which case AudioSwitch will switch over to emulating system/keyboard general VolumeUp and VolumeDown presses, for which I have no control over. I'll see what I can do and how it turns up working. Great idea indeed, I know that I need it now when I know what it is :)
